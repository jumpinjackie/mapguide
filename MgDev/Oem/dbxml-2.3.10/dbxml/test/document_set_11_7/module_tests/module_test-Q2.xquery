import module namespace test = 'http://test.module2' at 'module2.xquery';

test:function()
