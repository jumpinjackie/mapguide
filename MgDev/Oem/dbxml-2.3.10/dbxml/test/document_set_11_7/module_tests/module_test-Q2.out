<category name="Home" id="root">
  <category id="49-1127912717543" name="folder1"/>
  <category id="49-1127912719803" name="folder2"/>
</category><category id="49-1127912717543" name="folder1"/><category id="49-1127912719803" name="folder2"/>